tests/stubdata/output/wiley_jgra_57392.json

```diff
@@ -2,7 +2,7 @@
  "copyright": {"statement": "©2022. The Authors.", "status": true},
  "editorialHistory": {"acceptedDate": "2022-09-01", "receivedDates": ["2022-07-28"], "revisedDates": ["2022-08-26"]},
- "pagination": {"pageCount": "21"},
+ "pagination": {"electronicID": "e2022JA030882", "firstPage": "2022JA030882", "pageCount": "21"},
```

Comment:
I see that firstPage = the value in <id type="society" value="2022JA030882"/>. I'm just curious, is that whole string what we want to capture for first page? It seems more like an electronicID or something else?

Reply:
I didn't write the original code, so I'm not sure.  I suspect it's trying to work within the limits of the Ingest_Data_Model to capture multiple tags into the pagination structure.  I will need to ask Carolyn.